js/server/modules/@arangodb/foxx/manager.js

```diff
@@ -404,17 +410,20 @@ function propagateSelfHeal () {
   } catch (e) {
     console.errorStack(e, 'Failure during propagate self heal');
   }
-  reloadRouting();
 }
 
 // GLOBAL_SERVICE_MAP manipulation
 
 function initLocalServiceMap () {
   const localServiceMap = new Map();
   for (const mount of SYSTEM_SERVICE_MOUNTS) {
-    const serviceDefinition = utils.getServiceDefinition(mount) || {mount};
-    const service = FoxxService.create(serviceDefinition);
-    localServiceMap.set(service.mount, service);
+    try {
+      const serviceDefinition = utils.getServiceDefinition(mount) || {mount};
+      const service = FoxxService.create(serviceDefinition);
+      localServiceMap.set(service.mount, service);
+    } catch (e) {
+      console.errorStack(e);
```

Comment:
See above. These shouldn't normally throw errors anyway, but when they do we still want to try to carry on and not bring down everything.